packages/core/content-releases/admin/src/components/ReleaseListCell.tsx

```diff
@@ -0,0 +1,163 @@
+ * addColumnToTableHook
+ * -----------------------------------------------------------------------------------------------*/
+
+interface AddColumnToTableHookArgs {
+  layout: {
+    components: Record<string, CMAdminConfiguration>;
+    contentType: CMAdminConfiguration;
+  };
+  displayedHeaders: ListLayoutRow[];
+}
+
+const addColumnToTableHook = ({ displayedHeaders, layout }: AddColumnToTableHookArgs) => {
+  const { contentType } = layout;
+
+  if (!contentType.options?.draftAndPublish) {
+    return { displayedHeaders, layout };
+  }
+
+  return {
+    displayedHeaders: [
+      ...displayedHeaders,
+      {
+        key: '__release_key__',
+        fieldSchema: { type: 'string' },
+        metadatas: { label: 'To be released in', searchable: true, sortable: false },
+        name: 'releasedAt',
+        cellFormatter: (props: Entity) => <ReleaseListCell {...props} />,
+      },
+    ],
+    layout,
+  };
+};
+
+/* -------------------------------------------------------------------------------------------------
+ * ReleaseListCell
+ * -----------------------------------------------------------------------------------------------*/
+
+interface ReleaseListCellProps {
+  id: Entity['id'];
+}
+
+const ReleaseListCell = ({ id }: ReleaseListCellProps) => {
+  const releases = useReleasesList(id);
+  const [visible, setVisible] = React.useState(false);
+  const buttonRef = React.useRef<HTMLButtonElement>(null);
+  const { formatMessage } = useIntl();
+
+  const handleTogglePopover = () => setVisible((prev) => !prev);
+
+  return (
+    <Flex onClick={(e) => e.stopPropagation()}>
+      <Button type="button" onClick={handleTogglePopover} ref={buttonRef}>
+        <ActionWrapper height="2rem" width="2rem">
+          <Typography style={{ maxWidth: '252px', cursor: 'pointer' }} textColor="neutral800">
+            {releases.length > 0
+              ? formatMessage(
+                  {
+                    id: 'content-releases.content-manager.list-view.releases-number',
+                    defaultMessage: '{number} {number, plural, one {release} other {releases}}',
+                  },
+                  {
+                    number: releases.length,
+                  }
+                )
+              : '-'}
+          </Typography>
+          <Flex>
+            {releases.length > 0 && <SortIcon />}
+            {visible && (
+              <Popover
+                onDismiss={handleTogglePopover}
+                source={buttonRef as React.MutableRefObject<HTMLElement>}
+                spacing={16}
+              >
+                <ul>
+                  {releases.map(({ id, name }) => (
+                    <Box key={id} padding={3} as="li">
+                      {/* @ts-expect-error – error with inferring the props from the as component */}
+                      <Link to={`/plugins/content-releases/${id}`} as={NavLink} isExternal={false}>
```

Comment:
this isn't huge, but it's probably fine to use `Link` over `NavLink`, the latter has properties to handle when it's active, for the 99% of our use cases, we don't really need it.